modules/ROOT/pages/design-migrating-ms.adoc

```diff
@@ -0,0 +1,108 @@
+= Migrate to the Version 2 of the Mocking Service
+
+The mocking service lets you test endpoints that are defined in an API specification. The advantage to using the mocking service is that you do not need to implement the API before testing. To test an endpoint, the mocking service simulates calls to its methods.
+
+For example, when you are designing an API specification in API Designer, you can simulate a call to an endpoint’s GET method to see whether the response body for the 200 status code is as you want it. As a further example, if you are evaluating whether you want to implement an API specification that is in Exchange, you can simulate a call to an endpoint’s GET method to find out what the response body looks like.
+
+[IMPORTANT]
+====
+After January 31, 2020, version 1 of the mocking service will no longer send responses to requests. If you are using a third-party tool to send requests, the mocking service will return the HTTP status code 301 and provide a link to this document.
+====
+
+== How to Migrate an API-Specification Project to Version 2 of the Mocking Service
+
+If an API-specification project is older than January 10, 2019 and the mocking service was switched on before that date, follow these steps to migrate to version 2 of the mocking service:
```

Comment:
I think this sentence is a little too verbose, and the "…switched on before" portion was slightly confusing to me. Maybe something like:

> If the mocking service was switched on for a project before January 10, 2019, follow these steps to migrate to version 2:

Reply:
I agree with Tony. Also, it doesn matter if the project is older than January 10, 2019. That's redundant

Reply:
Good suggestion. Thanks.